### PR TITLE
Final configs

### PR DESCRIPTION
**Pull request by mastermind-mayhem**

### CODE REVIEW

File: src/main/java/frc/robot/team8410/commands/LowerIntakeCmd.java [hunk 1]
```diff
@@ -45,18 +45,18 @@ public void execute()
     
     if(currPOTVoltage >= 0.240) // put in constants
     {
-      speed = speed + .01;
-      if(speed >=.6)
-         speed = .6;
+      speed = speed + 0.01;
+      if(speed >=-0.6)
+         speed = -0.6;
```
Comment (mathyourlife) [inline src/main/java/frc/robot/team8410/commands/LowerIntakeCmd.java:50]: If this is reversing it should be starting at speed 0 and then decreasing by 0.01 until hitting a speed of -0.6.

File: src/main/java/frc/robot/team8410/commands/LowerIntakeCmd.java [hunk 2]
```diff
@@ -45,18 +45,18 @@ public void execute()
     
     if(currPOTVoltage >= 0.240) // put in constants
```
Comment (mathyourlife) [inline src/main/java/frc/robot/team8410/commands/LowerIntakeCmd.java:46]: The 0.240 and 0.270 should be swapped between the raise and lower intake commands.

**Comment by ashishusa** [comment]
Once we have the robot we can see what side are we rotating and then change the values to positive or negative 

On Mar 21, 2022, at 5:31 PM, Dan Couture ***@***.***> wrote:

﻿
@mathyourlife commented on this pull request.

In src/main/java/frc/robot/team8410/commands/LowerIntakeCmd.java:

> @@ -45,18 +45,18 @@ public void execute()
     
     if(currPOTVoltage >= 0.240) // put in constants
The 0.240 and 0.270 should be swapped between the raise and lower intake commands.

**Comment by mathyourlife** [comment]
Yup, agreed that the values will likely need to change.  My point was that
the values for high and low whatever they end up being should be swapped in
the motion profile checks.

On raise:
slowly ramp up to "low" value, stay constant until "high" value, slowly
ramp down speed until fully up

On lower:
slowly ramp up (in opposite direction) until the "high" value, stay
constant until "low" value, slowly ramp down until lowered completely

- Dan Couture

"What we have done for ourselves alone dies with us; what we have done for
others and the world remains and is immortal."
― Albert Pike

**Review by sallytheseal13 — APPROVED** [review]

**Pull request merged**